gradle-baseline-java/src/main/groovy/com/palantir/baseline/extensions/BaselineErrorProneExtension.java

```diff
@@ -50,6 +50,7 @@ public class BaselineErrorProneExtension {
             "LambdaMethodReference",
             "LoggerEnclosingClass",
             "LogsafeArgName",
+            "NonReusableAtlasdbPersister",
```

Comment:
Is this appropriate for this check? We want to move everyone to this new persister given the risks of the old persister